src/main/java/net/sf/jabref/preferences/PreviewPreferences.java

```diff
@@ -0,0 +1,108 @@
+package net.sf.jabref.preferences;
+
+import java.util.List;
+import java.util.Map;
+
+import net.sf.jabref.logic.citationstyle.CitationStyle;
+
+import static net.sf.jabref.Globals.prefs;
+
+
+public class PreviewPreferences {
+
+    private static final String CYCLE_PREVIEW_POS = "cyclePreviewPos";
+    private static final String CYCLE_PREVIEW = "cyclePreview";
+    private static final String PREVIEW_PANEL_HEIGHT = "previewPanelHeight";
+    private static final String PREVIEW_STYLE = "previewStyle";
+    private static final String PREVIEW_ENABLED = "previewEnabled";
+
+    private final JabRefPreferences preferences;
+
+
+    public PreviewPreferences(JabRefPreferences preferences) {
```

Comment:
See @oscargus recent PR's (for example https://github.com/JabRef/jabref/pull/1969) for how preference classes are structured (i.e. constructor takes a bunch of arguments describing the settings and `JabRefPreferences` handles the initialization and storage from/to the preferences.